jib-core/src/test/java/com/google/cloud/tools/jib/cache/LayerEntriesSelectorTest.java

```diff
@@ -63,10 +64,11 @@ public void testLayerEntryTemplate_compareTo() {
   }
 
   @Test
-  public void testToSortedJsonTemplates() {
+  public void testToSortedJsonTemplates() throws IOException {
     Assert.assertEquals(
         toLayerEntryTemplates(IN_ORDER_LAYER_ENTRIES),
-        LayerEntriesSelector.toSortedJsonTemplates(OUT_OF_ORDER_LAYER_ENTRIES));
+        ImmutableList.copyOf(
```

Comment:
iirc this test was failing because the `ArrayList` returned by `LayerEntriesSelector#toSortedJsonTemplates` wasn't equal to the `ImmutableList` returned by `toLayerEntryTemplates`, even though the contents were the same.

Reply:
Maybe we should just make `toLayerEntryTemplates` return `List`. Returning `ImmutableList` is too limiting without much value in tests.

Reply:
Hmm, that's weird - `ArrayList` should be equal to `ImmutableList` if contents are the same.